common/lib/capa/capa/templates/vsepr_input.html

```diff
@@ -21,13 +21,13 @@
    />
 
       <p class="status" aria-describedby="input_${id}">
-      ${status.display_name}
+         <span class="sr">${status.display_name}</span>
       </p>
 
   <p id="answer_${id}" class="answer"></p>
 
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="answer_molecule_select_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
"answer_molecule_select_${id}" doesn't appear elsewhere in this template. Is it a legitimate value to be using?